src/test/java/com/google/flourbot/entity/EntityModuleImplementationTest.java

```diff
@@ -0,0 +1,104 @@
+package com.google.flourbot.entity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.client.util.Charsets;
+import com.google.appengine.repackaged.com.google.common.io.Resources;
+import com.google.flourbot.datastorage.StubDataStorage;
+import com.google.flourbot.entity.action.sheet.SheetAppendRowAction;
+import com.google.flourbot.entity.action.sheet.SheetEntryType;
+import com.google.flourbot.entity.trigger.CommandTrigger;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EntityModuleImplementationTest {
+
+  private StubDataStorage dataStorage;
```

Comment:
it looks to me as a good use case for Mockito you can mock the data storage interface and then use when().thenReturn() pattern. Please check the other test PRs for some examples.

Reply:
Sure, I can look into replacing Stubs with Mockito!